src/main/java/io/github/mattidragon/universalperms/mixin/ServerPlayerEntityMixin.java

```diff
@@ -20,8 +20,12 @@ public abstract class ServerPlayerEntityMixin extends PlayerEntity {
     @Unique
     private boolean universal_perms$is_checking_permission;
 
-    public ServerPlayerEntityMixin(World world, BlockPos pos, float yaw, GameProfile gameProfile, @Nullable PlayerPublicKey publicKey) {
-        super(world, pos, yaw, gameProfile, publicKey);
+    // API call fails with publicKey --> Seems to have been removed from API
+    // public ServerPlayerEntityMixin(World world, BlockPos pos, float yaw,
+    // GameProfile gameProfile, @Nullable PlayerPublicKey publicKey) {
```

Comment:
Unnesssary comment. If the method changed, just update the mixin to reflect that, don't leave the old code behind